templates/ords-remix-jwt-sample/app/routes/venues.tsx

```diff
@@ -22,11 +22,11 @@ export const loader = async ({
 }: LoaderFunctionArgs) => {
   const userProfile = await auth.isAuthenticated(request);
   const USER_CREDENTIALS = userProfile === null
-    ? BASIC_SCHEMA_AUTH
+    ? null
     : `${userProfile.tokenType} ${userProfile.accessToken}`;
   const session = await getSession(request.headers.get('Cookie'));
   const error = session.get(auth.sessionErrorKey) as LoaderError;
-  const venues = await ORDSFetcher(`${VENUES_ENDPOINT}`, USER_CREDENTIALS) as ORDSResponse<Venue>;
+  const venues = await ORDSFetcher(`${VENUES_ENDPOINT}`, USER_CREDENTIALS!) as ORDSResponse<Venue>;
```

Comment:
could you please make evident that the user credentials parameter is now optional or it can be null in the `ORDSSFetcher` function? https://github.com/oracle/create-database-app/blob/597e88ed6454f6450b7a231ce490b22af36140cb/templates/ords-remix-jwt-sample/app/utils/ORDSFetcher.ts#L20 Otherwise we will still sending an `Authorization` header empty which is not something that we should be doing.

Reply:
Here is how the `headers` variable looks like when no authCredentials is = null:
`{ 'Content-Type': 'application/json', Accept: 'application/json' }`

And here is how it looks when we are logged in:
```
{
  'Content-Type': 'application/json',
  Accept: 'application/json',
  Authorization: 'Bearer <token_here>'
}
```

Reply:
Got it this works, just fine. I am not sold on still passing a null USER_CREDENTIALS to this function, we should make it clear that the endpoints we are accessing are unprotected from the app itself. Let's consider our options.